functions/js-bundle.js

```diff
@@ -3,7 +3,7 @@
 const createError = require("http-errors");
 const Raven = require("raven");
 const RavenLambdaWrapper = require("serverless-sentry-lib");
-const { jsBundle } = require("../src/jsBundle");
+const { jsBundle } = require("../src/js-bundle");
```

Comment:
should this file have a verb at the front too? like `create-js-bundle` or `bundle-js` ? (i notice a lot of the other ones start with a verb :))